### ISSUE TITLE
transition: GTFS import: Warn the user of any error occurring while importing

### ISSUE THREAD

**Issue by tahini** [chairemobilite/transition#258]
When any error occurs on the server, the current import operation should complete correctly, to have no lingering 'Operation in progress' in the interface, but the user should be warned that there was an error that prevents from importing the data, with some information as to what happens, so the user can act accordingly.

**Comment by samuel-duhaime** [comment]
@tahini Yes, I just had this issue, and I would have been lost without the help of @greenscientist . I had no idea; it was not working.
